src/main/java/Bai3.java

```diff
@@ -0,0 +1,20 @@
+public class Bai3 {
+    public String phanTichThuaSoNguyenTo(int number){
+        StringBuffer strResult = new StringBuffer("");
+        if(number == 1){
+            return "";
+        }
+        int jumbNumber = 2;
+        while (number != 1){
+            if(number % jumbNumber == 0){
+                strResult.append(jumbNumber + "*");
+                number/=jumbNumber;
+            }
+            else {
+                jumbNumber++;
+            }
+        }
+        int lenString = strResult.length();
```

Comment:
2 dòng này chỉ cần viết thành `return strResult.trim()` là xong

Reply:
2 dòng ni ý em là muốn xóa dấu '*' ở cuối, vì khi vòng for trả về thì chuỗi có dạng 1*2*3* 
còn phương thức trim() thì hắn chỉ giúp xóa khoảng trắng nên em chưa hiểu ý của anh.